bundles/configuration.rst

```diff
@@ -69,24 +76,28 @@ can add some configuration that looks like this:
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xmlns:acme-social="http://example.org/schema/dic/acme_social"
             xsi:schemaLocation="http://symfony.com/schema/dic/services
-                https://symfony.com/schema/dic/services/services-1.0.xsd">
-
+                https://symfony.com/schema/dic/services/services-1.0.xsd"
+        >
             <acme-social:config>
-                <acme-social:twitter client-id="123" client-secret="your_secret"/>
+                <acme-social:twitter client-id="123"
+                    client-secret="your_secret"
+                />
             </acme-social:config>
-
-            <!-- ... -->
         </container>
 
     .. code-block:: php
 
         // config/packages/acme_social.php
-        $container->loadFromExtension('acme_social', [
-            'twitter' => [
-                'client_id'     => 123,
-                'client_secret' => 'your_secret',
-            ],
-        ]);
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+        return static function (ContainerConfigurator $container) {
+            $container->extension('acme_social', [
+                'twitter' => [
+                    'client_id'     => 123,
+                    'client_secret' => 'your_secret',
+                ],
+            ]);
+        };
```

Comment:
@Nyholm any idea how to deal with this error?

Reply:
That error is the only one Im confident about. 

It just says that the "standard out-of-the-box symfony installation does not have a bundle that provides config for `acme`.  Which all make sense.

Ie, this error/warning is just to ignore.